Add README.md notice about update interval changes and issues (#303)
leikoilja/ha-google-home: README.md

local google assistant device (like Google Home/Nest etc) authentication
tokens and use those tokens making API calls to Google Home devices.

> ## IMPORTANT!!

Reviewer: ```suggestion
## IMPORTANT
```

With version 1.6 we have changed the update interval from 10 seconds to 180 seconds.

> This is due to this [#202](https://github.com/leikoilja/ha-google-home/issues/202) issue. This because of some people
> experincing devices timing out when trying to get data from them.

Reviewer: ```suggestion
This is due to [#202](https://github.com/leikoilja/ha-google-home/issues/202). Some people are
experiencing devices timing out when trying to fetch data from them.
```

> It is possible to manually change the update interval by pressing the `configure` button on the integration

Reviewer: ```suggestion
You can manually change the update interval by pressing the `configure` button on the integration
```

> page. Default is 180 seconds. Change this at your own risk!. If your devices are timing out,

Reviewer: ```suggestion
page. Default is 180 seconds. Change this at your own risk! If your devices are timing out,
```
